twingate/client.go

```diff
@@ -102,66 +82,110 @@ func (e *APIError) Error() string {
 }
 
 type MutationError struct {
-	Message string
+	Message graphql.String
 }
 
-func NewMutationError(message string) *MutationError {
+func NewMutationError(message graphql.String) *MutationError {
 	return &MutationError{
 		Message: message,
 	}
 }
 
 func (e *MutationError) Error() string {
-	return e.Message
+	return string(e.Message)
+}
+
+type HTTPClient interface {
+	Do(req *retryablehttp.Request) (*http.Response, error)
 }
 
 type Client struct {
-	APIToken         string
+	GraphqlClient    *graphql.Client
+	HTTPClient       HTTPClient
```

Comment:
There are 2 variables with the same name 
I think we can use the `retryablehttp` client inside the graphql lib , please check that